### ISSUE TITLE
Update this library on PyPI and surface link in README

### ISSUE THREAD

**Issue by dhermes** [arokem/python-matlab-bridge#105]
Is there a reason it isn't there?

Currently
```
pip install git+https://github.com/arokem/python-matlab-bridge/
```
"works just fine" but not all users have `git` installed.

On the flip side, most users without `git` are on Windows and this library doesn't (currently, as of Jan. 22, 15) support Windows.

**Comment by isbadawi** [comment]
It is there under the name `pymatbridge`. It's not the latest version though.

**Comment by dhermes** [comment]
Ahhh I see. I'll submit a PR to

1. Make this clear in the README
1. Add a Travis config to push to PyPI on new tags

Is this cool with the maintainers? (@isbadawi are you a maintainer?)

**Comment by isbadawi** [comment]
Nope, I just contributed a few times.

**Comment by arokem** [comment]
@dhermes - that does sound useful. The main reason I haven't been very good about uploading this to pypi is the very narrow bandwidth I have to devote to this project. 

On that note: @isbadawi - would you like to help maintain this code base, review code and merge PRs? You seem to care about this, and the code you've written has been helpful. I can make you a collaborator on the repo if you would like.

**Comment by arokem** [comment]
Speaking of Windows, there's a couple of nice write-ups here:

http://stackoverflow.com/questions/23716426/installing-pymatbridge-on-windows

About installing this on Windows. I don't have access to a machine with Windows, so I haven't been able to test this, but it does seem to be possible, with some effort.

**Comment by isbadawi** [comment]
Sure, I don't mind helping out with maintenance. I initially used this library for a part of my Master's thesis work which is now more or less wrapped up, so it's not as big a priority as it was then, but I'll do what I can to help.

**Comment by arokem** [comment]
Thanks! Welcome to the team...

**Comment by dhermes** [comment]
Closed by #114
